action.yml

```diff
@@ -91,6 +95,14 @@ runs:
           rm tailscale.tgz
           TSPATH=/tmp/tailscale_${VERSION}_${TS_ARCH}
           sudo mv "${TSPATH}/tailscale" "${TSPATH}/tailscaled" /usr/bin
+      - name: Link Tailscale binaries
+        shell: bash
+        if: ${{ inputs.download == 'false' }}
+        run: |
+          TAILSCALE_PATH=$(which tailscale)
+          TAILSCALED_PATH=$(which tailscaled)
+          sudo -E ln -s -f ${TAILSCALE_PATH} /usr/bin/tailscale
+          sudo -E ln -s -f ${TAILSCALED_PATH} /usr/bin/tailscaled
```

Comment:
does this work if the bins are already at that location?

Reply:
Good catch! I'll link conditionally

Reply:
Implemented in [5138375](https://github.com/tailscale/github-action/pull/121/commits/51383750648119e8c9231e0c5c150a20afb3b9f4)